### PR TITLE
cgltf: migrate to Conan v2

### PR DESCRIPTION
**Pull request by valgur**

### CODE REVIEW

**Comment by valgur** [comment]
Closing temporarily to avoid unnecessary load on the CI. Will reopen when I'm actively working on the PR again.

**Comment by RubenRBS** [comment]
dont break

File: recipes/cgltf/all/conanfile.py [hunk 1]
```diff
@@ -1,89 +1,88 @@
...
         )
 
+    def package(self):
+        copy(self, "LICENSE", dst=os.path.join(self.package_folder, "licenses"), src=self.source_folder)
+        cmake = CMake(self)
+        cmake.install()
+        for header_file in ["cgltf.h", "cgltf_write.h"]:
+            header_fullpath = os.path.join(self.package_folder, "include", header_file)
+            self._remove_implementation(header_fullpath)
+        for dll in (self.package_path / "lib").glob("*.dll"):
```
Comment (RubenRBS) [inline recipes/cgltf/all/conanfile.py:84]: You probably want to change this as you did for the other PRs

**Review by RubenRBS — CHANGES_REQUESTED** [review]
Thanks a lot! One request, the rest looks as good as this recipe will do with all the wonky things it does to the source

File: recipes/cgltf/all/conanfile.py (hunk 1 above)
Comment (valgur) [reply recipes/cgltf/all/conanfile.py:84]: For this one it was actually needed. Here's a CI run without it:
https://github.com/conan-io/conan-center-index/pull/18732#issuecomment-1746553400

The CMakeLists.txt should be tweaked to correctly handle this during the build, but simply copying the DLLs is sufficient. It at least avoids the need for additional patches.

**Review by RubenRBS — APPROVED** [review]

File: recipes/cgltf/all/conanfile.py (hunk 1 above)
Comment (RubenRBS) [reply recipes/cgltf/all/conanfile.py:84]: Thanks for the extra insight then! :)

**Review by jwillikers — APPROVED** [review]

**Pull request merged**